pkg/pillar/worker/opt.go

```diff
@@ -0,0 +1,41 @@
+package worker
```

Comment:
Missing copyright and license (might be the case for some other new files as well)

Reply:
Will add

Reply:
Done.